### PR TITLE
Update UG with edit commands, intro, glossary

### LINKED ISSUE #153: Add Glossary to UG

### LINKED ISSUE #157: Add or update Introduction and Target Users to UG, DG

### LINKED ISSUE #160: Add Get Edit Commands and Edit Commands to UG

### PR DESCRIPTION
**Pull request by oliviajohansen**

### CODE REVIEW

**Review by caitlinjee — APPROVED** [review]
lgtm

**Review by jiaqi20 — APPROVED** [review]
lgtm

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,7 +3,19 @@ layout: page
...
+## Introducing Wishful Shrinking
+Wishful Shrinking your desktop diet manager. It is an app that helps you manage your on-hand ingredients, organise personal recipes and track your diet. Wishful Shrinking facilitates a healthier diet in three main ways: 
+1. Provide a source of healthy, customizable recipes 
+2. Recommend recipes to improve ease of home cooking 
+3. Track daily food and calorie intake
+
+Wishful Shrinking targets busy office workers who tend to discount healthy eating. Office workers are also more familiar with desktop applications and typing and correspondingly, Wishful Shrinking is optimized for fast and efficient typers as it uses a Command Line Interface (CLI) with the added beauty of a Graphical User Interface (GUI).
+
+Choose a topic from the table of contents to get started on your Wishful Shrinking journey. 
+
```
Comment (TianYong-Goh) [inline docs/UserGuide.md:18]: Can consider moving this after the toc.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -260,6 +329,11 @@ _{give a remark to the recipe}_
 
 --------------------------------------------------------------------------------------------------------------------
 
+# Glossary
```
Comment (TianYong-Goh) [inline docs/UserGuide.md:333]: Can include the explanation of consumption also.

**Review by TianYong-Goh — APPROVED** [review]
I have left some suggestion. Other lgtm.

File: docs/UserGuide.md (hunk 2 above)
Comment (oliviajohansen) [reply docs/UserGuide.md:333]: Good idea ok

**Review by daongochieu2810 — APPROVED** [review]

**Pull request merged**